cognite/replicator/datapoints.py

```diff
@@ -90,10 +91,15 @@ def replicate_datapoints(
         latest_src_dp = Datapoints(timestamp=[src_datapoint_transform(latest_src_dp[0]).timestamp])
 
     _start, _end = _get_time_range(latest_src_dp, latest_dst_dp)
+
     if start is None:
         start = _start
+    else:
+        start = timestamp_to_ms(start)
     if end is None:
         end = _end
+    else:
+        end = timestamp_to_ms(end)

```

Comment:
Optional: Maybe would look better like below, but I am not sure.
`start = _start if start is None else timestamp_to_ms(start)`
`end =  _end if end is None else timestamp_to_ms(end)`